index.bs

```diff
@@ -3068,8 +3068,8 @@ The <dfn export for=commands>browsingContext.setViewport</dfn> command  specific
 
       browsingContext.SetViewportParameters = {
         context: browsingContext.BrowsingContext,
-        viewport: browsingContext.Viewport / null,
-        devicePixelRatio: (float .gt 0.0) / null,
+        ? viewport: browsingContext.Viewport / false,
```

Comment:
I prefer that we use null to reset emulation (to be consistent with the rest of the spec).

Reply:
also, changing to false is a breaking change

Reply:
@jgraham Because we use null for `?` as well, WDYT? I think semantically `false` implies "turn of emulation", so it makes sense as well. I don't have a strong opinion here, but we would need to use different wording for optional and null state then.

Reply:
We distinguish missing and null elsewhere in the spec, so if you're not able to make that distinction you're going to have a bad time (and we need to have a broader conversation about the API conventions).